src/Filesystem/FileList.php

```diff
@@ -57,6 +66,15 @@ public function getInitCloudLog(): string
         return $this->directoryList->getInit() . '/var/log/cloud.log';
     }
 
+    /**
+     * @return string
+     * @throws UndefinedPackageException
+     */
+    public function getInitCloudErrorLog(): string
+    {
+        return $this->directoryList->getInit() . '/var/log/cloud.error.log';
```

Comment:
fore retrieving path for logs you may use `$this->directoryList->getPath(DirectoryList::DIR_LOG, true)`

Reply:
Thanks, done